pkg/ccl/crosscluster/logical/lww_row_processor.go

```diff
@@ -291,7 +300,7 @@ func (srp *sqlRowProcessor) ProcessRow(
 	}
 
 	var stats batchStats
-	if row.IsDeleted() {
```

Comment:
because we can no longer rely on `row.IsDeleted()`,  delete entries will appear in the DLQ as updates/inserts :( 
https://github.com/msbutler/cockroach/blob/butler-move-origin-id/pkg/ccl/crosscluster/logical/dead_letter_queue.go#L177

Reply:
Hrmmmmmm.  Well. I pushed a bad idea. But I'm kinda thinking we just take the part of this that fixes the single column row and leave the other bits for now for 24.2

Reply:
Agreed, I think we can live with this known limitation for 24.2